src/lib.rs

```diff
@@ -1820,6 +1820,57 @@ pub mod __rt {
         }
     }
 
+    pub mod utils {
+        pub mod consts {
```

Comment:
I don't think these nested submodules are particularly necessary, putting these functions directly into `__rt` is fine.

Reply:
Ok